Tests/XcodeprojTests/FunctionalTests.swift

```diff
@@ -147,8 +147,10 @@ func XCTAssertXcodeBuild(project: AbsolutePath, file: StaticString = #file, line
         if env["TOOLCHAINS"] == nil {
             env["TOOLCHAINS"] = "default"
         }
+        let schemeName = String(project.basename.dropLast(10)) + "-Package"
+        let buildDir = project.parentDirectory.appending(component: "build").asString
         try Process.checkNonZeroExit(
-            args: "xcodebuild", "-project", project.asString, "-alltargets", environment: env)
+            args: "xcodebuild", "-project", project.asString, "-scheme", schemeName, "-derivedDataPath", buildDir, environment: env)
```

Comment:
I ended up adding manifest reference for all dependencies because it will be non trivial to compute edited dependencies outside of `Workspace`. I think it should be fine to add manifest references of non-edited packages as we add their source files anyway.